AppClient.php

```diff
@@ -0,0 +1,63 @@
+<?php
+    use GuzzleHttp\Client;
+    use GuzzleHttp\HandlerStack;
+    include('ClientLogger.php');
+    include('Stubber.php');
+    include('CacheStack.php');
```

Comment:
You can use autoloader in the same way as in other files:
`require 'vendor/autoload.php';`